.github/workflows/grid_client_nightly.yml

```diff
@@ -91,6 +91,23 @@ jobs:
         run: |
           yarn run ts-node --project packages/grid_client/tsconfig-node.json packages/grid_client/scripts/delete_all_contracts.ts
 
+      - name: Run check up - List all contracts
+        id: command_step
+        run: |
+          sleep 15
+          yarn run ts-node --project packages/grid_client/tsconfig-node.json packages/grid_client/scripts/list_all_contracts.ts > output.txt
+
+      - name: Check if contracts are Empty
+        run: |
+          # Read the Contracts line from the file
+          contracts_json=$(awk 'NR==5' output.txt)
+
+          # Check if all three [Name - Node - Rent] contracts are empty
+          if [[ $contracts_json != *'nodeContracts: []'* ]]; then
```

Comment:
The comment mentions that it checks for `[Name - Node - Rent]` while the condition itself seems to check nodeContracts only.